cips/cip-draft_ica.md

```diff
@@ -0,0 +1,917 @@
+- `Interchain Account`: An account on a host chain. An interchain account has all the capabilities of a normal account. However, rather than signing transactions with a private key, a controller chain will send IBC packets to the host chain which signals what transactions the interchain account must execute. 
+- `Interchain Account Owner`: An account on the controller chain. Every interchain account on a host chain has a respective owner account on the controller chain. 
+
+The IBC handler interface & IBC relayer module interface are as defined in [ICS-25](../../core/ics-025-handler-interface) and [ICS-26](../../core/ics-026-routing-module), respectively.
+
+### Desired properties
+
+- Permissionless: An interchain account may be created by any actor without the approval of a third party (e.g. chain governance). Note: Individual implementations may implement their own permissioning scheme, however the protocol must not require permissioning from a trusted party to be secure.
+- Fault isolation: A controller chain must not be able to control accounts registered by other controller chains. For example, in the case of a fork attack on a controller chain, only the interchain accounts registered by the forked chain will be vulnerable.
+- The ordering of transactions sent to an interchain account on a host chain must be maintained. Transactions must be executed by an interchain account in the order in which they are sent by the controller chain.
+- If a channel closes, the controller chain must be able to regain access to registered interchain accounts by simply opening a new channel.
+- Each interchain account is owned by a single account on the controller chain. Only the owner account on the controller chain is authorized to control the interchain account. The controller chain is responsible for enforcing this logic.
+- The controller chain must store the account address of any owned interchain accounts registered on host chains.
+- A host chain must have the ability to limit interchain account functionality on its chain as necessary (e.g. a host chain can decide that interchain accounts registered on the host chain cannot take part in staking).
+
+### General design 
+
+A chain can utilize one or both parts of the interchain accounts protocol (*controlling* and *hosting*). A controller chain that registers accounts on other host chains (that support interchain accounts) does not necessarily have to allow other controller chains to register accounts on its chain, and vice versa. 
+
+This specification defines the general way to register an interchain account and send tx bytes to be executed on behalf of the owner account. The host chain is responsible for deserializing and executing the tx bytes and the controller chain must know how the host chain will handle the tx bytes in advance of sending a packet, thus this must be negotiated during channel creation.
+
+### Controller chain contract
```

Comment:
@sampocs @womensrights should we remove the discussion of the controller? since Celestia won't be adding it, it doesn't seem relevant / distracts from the minimal change that we do want to make (add host functionality)

Reply:
My thinking was that the controller module provides important context for the full ICA solution - something that the team would probably be interested in, even if they're just adding the host module (made a note [here](https://github.com/Stride-Labs/celestia-CIPs/pull/1/files#diff-44ed92b035a9afbbf3dde6d410353a4bbd872168949a32a73a34d19aee5db806R22)).

That said, it probably doesn't need to have as much detail as it currently does

Reply:
I agree its still important context. I think given its still in the draft stage, I would probably leave it in then get feedback during the core calls / forum as to whether it should be omitted